core/src/xyz/magicjourney/nebulaquest/entity/entities/Start.java

```diff
@@ -8,6 +8,8 @@
 import xyz.magicjourney.nebulaquest.player.Player;
 
 public class Start extends Entity {
+  private static final int MONEY_ON_PASS_START = 200;
+
   public Start() {
```

Comment:
Zamień proszę wartość 200 w tym napisie na zmienną MONEY_ON_PASS_START, ułatwi nam to później balansowanie rozgrywki. Zmiana zarabianych przez gracza pieniędzy ograniczy się do zmiany jednej zmiennej.